container/containerd/client.go

```diff
@@ -48,6 +49,7 @@ var ctrdClient ContainerdClient = nil
 
 const (
 	maxBackoffDelay   = 3 * time.Second
+	baseBackoffDelay  = 100 * time.Millisecond
```

Comment:
Where did this number come from?

Reply:
I had to set it to some meaningful value and I thought that 100ms might make sense.